notary-api/src/routes/selles.js

```diff
@@ -0,0 +1,39 @@
+import express from 'express';
+import { asyncError } from '../utils';
+import { saveSeller } from '../services/sellerService';
+
+const router = express.Router();
+
+/**
+ * @swagger
+ * /sellers/heads-up:
+ *   post:
+ *     description: |
+ *       Endpoint where the WAPI will make the POST request when the registration is finished.
```

Comment:
I'd put: `Endpoint where the sellers registrerer will send the information regarding a new seller`